docs/contributing.md

```diff
@@ -78,9 +79,11 @@ Check linting rules are passing:
 ```shell
 make checks-validator
 ```
-
 lakeFS uses [go fmt](https://golang.org/cmd/gofmt/) as a style guide for Go code.
 
+Created a system test as part of the change? Want to ensure our CI job passes before pushing the changes remotely? Use [this](https://github.com/treeverse/lakeFS/blob/master/esti/README.md)  guide to enable local execution of our system tests
```

Comment:
@adipolak @peacing 
Would you mind reviewing the contribution docs?

Reply:
@adipolak  @peacing Your review on the new README will also be much appreciated :)